packages/nimble-components/src/table/components/group-row/index.ts

```diff
@@ -30,17 +38,35 @@ export class TableGroupRow extends FoundationElement {
     @attr({ mode: 'boolean' })
     public expanded = false;
 
+    @attr({ mode: 'boolean' })
+    public selectable = false;
+
+    @attr({ attribute: 'selection-state' })
+    public selectionState: TableRowSelectionState = TableRowSelectionState.notSelected;
```

Comment:
Do you think using the mixin pattern for this implementation makes sense, since we replicate it in 3 places?

Reply:
I'm not sure. I lean towards saying it's not worth it, at least for now, partially because the pattern is slightly different on the row. It has a boolean to indicate whether or not it's selected rather than a `TableRowSelectionState` property, and I'd prefer not to change that.